anyblok/plugins.py

```diff
@@ -51,8 +51,6 @@ class AnyBlokPlugin(Plugin):
 
     def __init__(self):
         super(AnyBlokPlugin, self).__init__()
-        self.authoried_bloks_test_files = []
```

Comment:
Here I simply prefer getting AttributeError if we introduce a bug and they don't get initialized in some code path, rather than debugging for hours to understand why no tests are launched.

Reply:
I'd even go as far as using class attributes for the default values for 'registryLoaded' and 'AnyblokOptions', so that we don't even need to implement `__init__` and to risk incompatibilites etc.